pennylane/ops/qubit/parametric_ops.py

```diff
@@ -26,11 +26,13 @@
 import pennylane as qml
 from pennylane.operation import AnyWires, Operation
 from pennylane.ops.qubit.non_parametric_ops import PauliX, PauliY, PauliZ, Hadamard
-from pennylane.utils import expand, pauli_eigs
+from pennylane.operation import expand_matrix
```

Comment:
Perhaps this allows us to remove `pennylane.utils.expand`?

Reply:
Ah, some background - `pennylane.utils.expand` was added a long time ago, as a utility function to expand statevectors/matrices. It is non-differentiable.

@mariaschuld added `pennylane.operation.expand_matrix`, as it was easier to write a new function that was autodiff compatible rather than modify the existing one.

With this change, if `pennylane.utils.expand` is no longer used anywhere, we could create an issue to remove it.

Reply:
Done #2637